packages/react-integration/demo-app-ts/src/components/demos/DescriptionListDemo/DescriptionListDemo.tsx

```diff
@@ -29,7 +29,7 @@ export class DescriptionListDemo extends Component {
         <Divider component="div" />
         <br />
         <div className="example">
-          <DescriptionList id="simple-description-list">
+          <DescriptionList id="simple-description-list" aria-label="Basic description list">
```

Comment:
Let's update these demos to have similar aria-labels as the example/demo files from the react-core package (i.e. remove "list" or "description list" from the labels etc)